apps/web/src/services/dashboards/components/dashboard-folder/DashboardFolderTreeItem.vue

```diff
@@ -59,6 +62,11 @@ const state = reactive({
             },
         ];
     }),
+    slicedLabels: computed(() => props.treeData.data?.labels?.slice(0, 3) || []),
+    showMoreLabels: computed(() => {
+        if (!props.treeData.data.labels) return false;
+        return props.treeData.data.labels.length > LABELS_LIMIT;
+    }),
```

Comment:
suggestion: I think there must be null check too in showMoreLabels computed as like slicedLabels.